includes/sanitizers/class-amp-iframe-sanitizer.php

```diff
@@ -142,11 +142,43 @@ public function sanitize() {
 
 			$new_node = AMP_DOM_Utils::create_node( $this->dom, 'amp-iframe', $normalized_attributes );
 
-			if ( true === $this->args['add_placeholder'] ) {
-				$placeholder_node = $this->build_placeholder( $normalized_attributes );
+			// Find existing placeholder/overflow.
+			$placeholder_node = null;
+			$overflow_node    = null;
+			foreach ( iterator_to_array( $node->childNodes ) as $child ) {
+				if ( ! ( $child instanceof DOMElement ) ) {
+					continue;
+				}
+				if ( $child->hasAttribute( 'placeholder' ) ) {
+					$placeholder_node = $node->removeChild( $child );
+				} elseif ( $child->hasAttribute( 'overflow' ) ) {
+					$overflow_node = $node->removeChild( $child );
+				}
+			}
+
+			// Add placeholder.
+			if ( $placeholder_node || true === $this->args['add_placeholder'] ) {
+				if ( ! $placeholder_node ) {
+					$placeholder_node = $this->build_placeholder( $normalized_attributes ); // @todo Can a better placeholder default be devised?
+				}
 				$new_node->appendChild( $placeholder_node );
 			}
 
+			// Add overflow.
+			if ( $new_node->hasAttribute( 'resizable' ) && ! $overflow_node ) {
+				$overflow_node = $this->dom->createElement( 'button' );
+				$overflow_node->setAttribute( 'overflow', '' );
+				if ( $node->hasAttribute( 'data-amp-overflow-text' ) ) {
+					$overflow_text = $node->getAttribute( 'data-amp-overflow-text' );
+				} else {
+					$overflow_text = __( 'Show all', 'amp' );
+				}
+				$overflow_node->appendChild( $this->dom->createTextNode( $overflow_text ) );
+			}
+			if ( $overflow_node ) {
```

Comment:
Is the `overflow` element still needed if `amp-iframe` does not have the `resizeable` attribute?

Reply:
The `overflow` element doesn't do anything if the `amp-iframe` is not `resizable`, but including it is not an error.

Reply:
As far as I understand from the amp-iframe docs, `overflow` is only used for resizable, so probably no need to copy it over if `resizable` isn't set.

Reply:
It doesn't make sense for the `overflow` to be tucked under the `noscript > iframe` element even less, so I think moving it to be a child `amp-iframe` even if it is not `resizable` is good.

Reply:
Agreed, just wanted to make sure that's the intended way.

Reply:
Works for me. Thanks!